### DISCUSSION TITLE
[Balance] Clothes should not have resistances (reasons inside)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#5339] [Unstable feedback]
A often overlooked aspect of balance is how player Gear changes resistances: Some clothes resist types of damage more than others. With the addition of the Combat Suit, the difference in effective HP are quite staggering, so let's talk about it

According to: **https://barotraumagame.com/wiki/Gear** _Note: (updated wiki link, values are still the same)_

The obvious issues here looking at the gear page are, by order of importance: 

1. **Equipping Helmets is a no-brainer**. Having a helmet beats having any other type of regular headgear. You want the entire crew to wear a helmet at all times, _even under a diving suit_, with two exceptions: medic goggles and diving mask. 

2. **Body armor has no competition**.  You want a fully body-armored crew at all times, because the only other item that goes in the body slot is the diving suit (and the rare moloch shell). There is no reason to not walk around in body armor when in dry environment

3. **Toolbelt has no competition**. _Nothing_ else goes in the toolbelt slot. This slot is literally dedicated to one item, the toolbelt. Can even be worn on top of a diving suit. While inventory space is a hot topic, there's a definitive oversight here

4. **There are only two good types of clothing: Security and Medic (A)**. The tradeoff is resistance vs 3 inventory slots. 
Captain clothes have *nothing*, Engineer, Mechanic and Assistant have very minor resistances, but Security has everything. Even between medic clothes, one has resistances and the other doesn't. That means, again, that you want the entire crew to use Security clothes for the massive benefit of taking less damage from every source, _especially when diving_

5. **Skills' success rate reduces the need for resistances**.  In the case of the one good engineer clothing, the Jumpsuit, which gives +60% burn resistance, this type of clothing is better suited to the mechanic, and vice-versa! Burns are more common on classes with low electrical skill, as **they will fail to repair electrical items more often**. There are also way more ELECTRICAL repairs than MECHANICAL repairs, leading in a much higher ratio of burns than lacerations. As a result, Mechanics don't want mechanic clothes; they want either Jumpsuits or Security clothing, while Engineers don't need that much burn resist and would be happier with Sec clothing. _Note: Mechanic clothes don't even have Laceration resistances, making them worse than engineer clothes as well, why?_

6. **Security spawns with too much of everything**. This is more of an economic issue, but security spawns with the best clothes as well as a free body armor and helmet; no need to actually buy equipment through a campaign, as you can easily collect free spare armor and helmets from players. Commando in particular has a staggering amount of powerful items.

7. **Health scanner is free to use**. A minor balance issue, but health scanner is by definition better than any headwear that isn't helmet. Unless seeing medical condition and the small green HUD bothers you, you can equip the scanner at all times unlike the Diving Mask

----

The inherent imbalance in clothing gear means there is an optimal way of playing that involves minmaxing clothing: Some clothes are inherently worse than others, and it pushes players to discard their class clothing almost permanently the "better" ones; especially Captains, Mechanics and Assistants. 

# How to balance clothing? Separating clothing into two types: Cosmetic clothing, and Functional clothing

* **All basic clothing should be purely cosmetic**, with no resistances applied, and no inventory slots, existing only for looks. This would eliminate the balance issues between classes.

* **Functional clothing should be using the armor/head/belt slot**. Body armor, helmet, but also Toolbelts need competition in their respective slots for FUNCTIONAL tradeoffs: resistances, inventory slots, or other types of utility. Rather than a single body armor protecting from ALL types of damage, several "body/helmet" equipments covering a specific purpose: 
Kevlar vest and Ballistic Helmet against bullets/piercing, Hazmat suit against radiation/gas, Body armor against bites and lacerations, more toolbelt alternatives, **or even going as far as** making the Toolbelt use the "body" slot and putting the Headset in the same slot as helmets. **Gear slots needs to have an opportunity cost**, between other items in the same slots, or no item at all.

Additional suggestions:

* Some resistances should be attached to skill rather than cloting. Partial Burn and Laceration resistances to Electrical and Mechanical skill, Bite resistance to combat skill...

**Comment by Likvidus** [comment]
Dividing the gear into cosmetic and functional is a good suggestion. But, in my opinion, this will deprive the classes of their individuality. Some of my suggestions:
1. Helmets - cannot be worn under a diving suit, reduces the player's speed by 10%, reduces the viewing range?
2. Body armor - reduces the player's speed by 20-30%.
3. Belts - add alternative options - doctor's bag, ammo belt, backpacks and others. We'll have to balance the original belt to make it all work.
4. Health Scanner - add consumable battery.

**Comment by CapoFantasma97** [comment]
Engineer, Mechanic, have resistences for their jobs (radioactive poisoning, burns, physical trauma related to failed works).
I'd rather wear that as engineer/mechanic, than security attire. Resistance to enemy caused damage is not as important if you stay on the ship. If anything, I'd make sure to have uniforms properly balanced and specialized, rather than removing resistances.

I don't see the problem of toolbelts/body armor/helmets not having a "real competition" _for now_. If you have marks to spare, of course you'd want to gear up your crew! But do keep in mind that they also slow down the character a bit, so the ones less exposed to enemies (mantainers not involved with diving teams or with security) could live without. And you can also wear the flashlight as headgear, just an heads up if you need your hands free while working in a dark environment.

**Burns and lacerations are more common for the people that actually work in maintenance**. No class starts out at level 55, which is the only thing ensuring you'll never get a work related accident. As an engineer, I have to occasionally get my burns checked by a medic, and the jumpsuit having a burn resistance helps with that too, otherwise damage would be higher. I don't want to make my character sleep for hours or waste medical supplies. If the commander/gunner/medic has a broken panel or gun, they call the engineers/mechanics to fix that, they don't do that themselves. Work safety first! Just like IRL.

While health scanner doesn't drain energy, it's also very limited in quantity (only medics spawn with those,  and I don't think you could buy them). You also don't need it in most situations, it helps for medics to take a glance and know what meds to take, rather than going back and forth just to check. When someone is in danger, the ship's medics are always called, what others could and should do as first aid is to bandage bleeding wounds/do CPR/put oxygen mask on the victim.
Also, headsets used to drain energy. It was just an annoying inconvenience, it didn't make them any less powerful nor it balanced them in any meaningful way. Same could apply to health scanners.

Eventually, more content will be implemented with the game, gadgets and wearables useful for other stuff and classes.

**Reply by Krzeszny** [reply]
Sorry but I have to destroy **some** of your arguments.

Engineer does have the resistance for their job but the nuclear engineer's hazmat suit trades 50% of the burn/electrical resistance for radiation resistance and there are normally no radiation hazards in Baro if you're not playing the mod Into The Abyss. 
If the reactor explodes, it's practically only due to griefers and the hazmat suit won't likely save you, so it's the inferior overall.

Secondly, you _wouldn't_ (if you were informed) rather wear mechanic's clothes as a mechanic because guess why? Mechanic's clothes protect from burns/electrical hazards - **but much less than engineer's clothes.** Mechanic's clothes are a worse version of electrical engineer's clothes (40% vs 60% burns protection). Security clothes protect from all hazards, including mechanical and electrical hazards, so in the current state of the game you'd want to wear security clothes as a mechanic. But then, as you level up enough to repair JB's and machines without risking damage, the job clothes will become inferior to security clothes if not useless.
Thirdly, in multiplayer not only security officers go for diving missions.

**Comment by CapoFantasma97** [comment]


**Reply by Krzeszny** [reply]
Dude! Why?! You've posted a link to the **unused AND outdated** wiki. uberpendragon posted the right link and your counterargument is a link to the wrong wiki and an assumption that the official wiki is outdated.

**Reply by ubertpendragon** [reply]
> Mechanic's clothing (both variants) have: 50 gunshot, 50 blunt force, 40 burn, 50 lacerations (if the page isn't outdated)

The misconception in that page comes from added INNATE human resistance on top of clothes
![image](https://user-images.githubusercontent.com/73229309/112810115-04bffc00-907b-11eb-898f-7d230b587984.png)

The boiler suit only gives 40 burn resistance on top of innate 50 50 50 resistances 

But yes, I linked the wrong wiki; clothing values are the exact same though

**Comment by Tsunder** [comment]
2. body armor's competition is an empty body slot - an empty slow allows diving suits to be instantly put on without requiring an empty slot in the main inventory to hold the body armor. that's not saying much though, I suppose. Diving suit is also somewhat useful to be walking around in, because that means one's always ready for the pressure.

re: response
form is function

medic garb doing medic-y things seems fine, along gear in general providing resistences.

cosmetic-fying clothing homogenizes classes further.

while skills can get very high in a safe long running campaign, missions and other causes of low level (death) help keep starting skills/gear relevant.

clothes should amplify the class's identity, I think, while still being not useless to other classes.

wiki has moved FYI: https://barotraumagame.com/wiki/Gear

**Reply by adamkad1** [reply]
So, have clothing boost relevant skills?

**Comment by hnappinn** [comment]
1. Clothing, armor and diving suits should wear out. 
2. General armor should not provide very good protection.  
3. We need specialized armor from bullets, from blows / fire / cuts / bites, from explosions / shrapnel. General armor will protect against many things, but worse than specialized armor. 
4. The slowdown will depend on the type of armor

**Reply by StruggleDuck** [reply]
I disagree, It would not make sense for armor to only be good against certain damage types, and feel that armor is in a good place currently. on top of that, with the next update, there is planned to be 2 items that would help provide protection for workplace hazards, one of them being the safety harness (I think), and the PUCS. both provide protection against more workplace related damages, and take up the diving suit slot.
